src/lib/middleware/getServerSideProps.ts

```diff
@@ -61,7 +61,7 @@ export const getServerSideProps: GetServerSideProps<ServerSideProps> = async (ct
       user_registration: config.features.user_registration,
       oauth_registration: config.features.oauth_registration,
       oauth_providers: JSON.stringify(oauth_providers),
-      bypass_local_login: config.oauth.bypass_local_login,
+      bypass_local_login: config.oauth?.bypass_local_login ?? false,
```

Comment:
Shouldn't the `config.oauth.bypass_local_login` have a default value within the config?

Reply:
When no oauth configs are set, it's null.

Reply:
However if at least one oauth config is present, then the defaults for the others will apply.